src/jupyter_contrib_nbextensions/nbextensions/toc2/toc2.yaml

```diff
@@ -32,5 +32,16 @@ Parameters:
   description: Display Table of Contents as a navigation menu
   input_type: checkbox
   default: true
-  
+- name: toc2.colors.hoover_highlight
+  input_type: color
+  description: Hoover color in toc
+  default: ["#DAA520"]  
```

Comment:
Yep. I asked myself, but did not investigated since it worked immediately as such.